webpack.config.js

```diff
@@ -2,7 +2,7 @@ const LiveReloadPlugin = require('webpack-livereload-plugin')
 const isDev = process.env.NODE_ENV === 'development'
 
 module.exports = {
-  entry: './client/index.js',
+  entry: ['babel-polyfill', './client/index.js'],
```

Comment:
Forces `babel-polyfill` to be included in the bundle. Not exactly an intuitive change to go from a single string to an array of strings. Also mixes file names and package names.